docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/listing/list_item_macros.twig

```diff
@@ -421,14 +418,7 @@
     {% if media_type == "Video" or media_type == "Infographic" %}
       <span class="cgdp-media-link__mediatype">({{ media_type }})</span>
     {% elseif media_type == "File" %}
-      <span class="cgdp-media-link__filetype {{file_type}}">
-        {% if file_type != "unknown" %}
-          {# Note, the spanish should have the word order reversed... #}
-          <span class="usa-sr-only">{{ file_type }} {% trans %}file{% endtrans %}</span>
-        {% else %}
-          {{ file_ext }}<span class="usa-sr-only"> {% trans %}file{% endtrans %}</span>
-        {% endif %}
-      </span>
+      <span> {{file_type}} </span>
       <span class="cgdp-media-link__filesize">({{ file_size }})</span>
     {% endif %}
```

Comment:
This one is just wrong in that it is using the old filesize and not wrapping the whole thing? https://github.com/NCIOCPL/cgov-digital-platform/pull/4903#discussion_r2325310192